gatk-preprocess.wdl

```diff
@@ -1,41 +1,43 @@
-import "tasks/gatk.wdl" as gatk
 import "tasks/biopet.wdl" as biopet
+import "tasks/gatk.wdl" as gatk
 import "tasks/picard.wdl" as picard
 
 workflow GatkPreprocess {
     File bamFile
     File bamIndex
     String outputBamPath
-    File ref_fasta
-    File ref_dict
-    File ref_fasta_index
+    File refFasta
+    File refDict
```

Comment:
full `referenceFasta` and `referenceDict` is preferable in my opinion, but we might need to discuss this also with @ffinfo.